FE/src/pages/Menu/Menu.jsx

```diff
@@ -118,13 +96,15 @@ function parseData(kidData) {
 }
 
 function removeCityPrefix(address) {
-    return address.replace("서울특별시 ", "");
+    return address !== null ? address.replace("서울특별시 ", "") : "null";
```

Comment:
```suggestion
return address?.replace("서울특별시", "")
```
이렇게 쓰면 이해가 더 빠를 것 같아요.